bin/nib-ls

```diff
@@ -180,6 +184,22 @@ def proc_file(f, opts):
     else:
         row += [ '' ]
 
+    if opts.header_fields:
+        # signals "all fields"
+        if opts.header_fields == '*':
```

Comment:
Is this the right character to use?   Maybe 'all' string instead to avoid shell?